.gitignore

```diff
@@ -1,6 +1,6 @@
 # Output directory for compiled TS
 lib
-types
+/types
```

Comment:
is this for not indexing types at the root level but keeping those under `src`?

Reply:
Exactly 👍